src/seml/database.py

```diff
@@ -211,7 +231,9 @@ def build_filter_dict(filter_states, batch_id, filter_dict, sacred_id=None):
     return filter_dict
 
 
-def get_max_in_collection(collection, field: str):
+def get_max_in_collection(
```

Comment:
Making return type `int` should solve an issue above?

Reply:
I changed the function to accept an type